pkg/csconfig/console.go

```diff
@@ -15,16 +15,18 @@ const (
 	SEND_CUSTOM_SCENARIOS  = "custom"
 	SEND_TAINTED_SCENARIOS = "tainted"
 	SEND_MANUAL_SCENARIOS  = "manual"
+	RECEIVE_DECISIONS      = "decisions_poll"
```

Comment:
doesn't `decisions_sync` make more sense ?

Reply:
yes this was my first thought too. I'm afraid it will lead to confusion given the current behaviour of "send_manual_scenaruios" (that sends "manual_decisions"). We can change it, but let's keep in mind backward compat. What do you suggest ?